vendors/nordic/boards/nrf52840-dk/aws_tests/application_code/main.c

```diff
@@ -276,6 +276,15 @@ static void prvUartInit( void )
     APP_ERROR_CHECK( xErrCode );
 }
 
+/**
+ * @brief Initialize BLE stask for Nordic.
+ * On Nordic it is just a stub.
```

Comment:
Dont we have vendor specific stack initialization for Nordic ?

Reply:
No, it comes with softdevice